pls check now

florencered opened the pull request.

NikhilC2209 commented:
no need for 2 nested folders, keep all contents inside cyber-defender and get rid of cyber-defender-main.
Also update the paths wherever necessary.

florencered commented:
As asked I have made the following commits:
1.Deleted the cyber defender main folder previously nested inside cyber defender folder
Now the cyber defender file contains all the necessary code.
Pls check,thanks

florencered commented:
As asked I have made the following commits:
1.Deleted the cyber defender main folder previously nested inside cyber defender folder
Now the cyber defender file contains all the necessary code.
2.updated the paths
All the code is up to date
Pls check,thanks
